scriptworker/test/__init__.py

```diff
@@ -236,20 +236,6 @@ def integration_create_task_payload(config, task_group_id, scopes=None,
     }
 
 
-@pytest.yield_fixture(scope='function')
```

Comment:
+1 on testing the other scripts. At least `beetmoverscript` and `bouncerscript` are heavily using `event_loop`. Am happy to help with removing the same way we did here in a subset of the *scripts if you want to divide work. I can take care of {balrog,beetmover,shipit,bouncer}scripts.